cmd/mr_test.go

```diff
@@ -63,6 +71,13 @@ func Test_mrCmd(t *testing.T) {
 	var mrID string
 	t.Run("prepare", func(t *testing.T) {
 		cleanupMR(t, "lab-testing", repo, "mr title")
+		cleanUpCmd := exec.Command(labBinaryPath, "mr", "close", "4676")
```

Comment:
Was this step added only for closing that MR that was causing the build to fail?
Closing an MR with a hard-coded number doesn't seem correct to be kept.